go.mod

```
@@ -11,7 +11,7 @@ require (
 	github.com/cespare/cp v1.0.0
 	github.com/clevergo/websocket v1.0.0
 	github.com/davecgh/go-spew v1.1.1
-	github.com/dgraph-io/badger v1.6.0
+	github.com/dgraph-io/badger/v3 v3.2011.1
```

Couldn't this be without the `v3`? Specifying the major version is useful when using several major versions a t the same time.
```suggestion
	github.com/dgraph-io/badger v3.2011.1
```

---

`/V3` is needed because v3.2011.1 is not compatible with v0 and v1 major version!

---

@winnie-byun thanks for the reference